src/components/workflowEditor/NoGitOpsRepoConfiguredWarning.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { FunctionComponent, useState } from 'react'
+import { useHistory } from 'react-router-dom'
+import { VisibleModal2 } from '@devtron-labs/devtron-fe-common-lib'
+import { URLS } from '../../config'
+import { ReactComponent as WarningIcon } from '../../assets/icons/ic-warning.svg'
+import { ReactComponent as Close } from '../../assets/icons/ic-cross.svg'
+import { NoGitOpsRepoConfiguredWarningType, ReloadNoGitOpsRepoConfiguredModalType } from './types'
+import { ReactComponent as ArrowRight } from '../../assets/icons/ic-arrow-right.svg'
+import { ReactComponent as RetryIcon } from '../../assets/icons/ic-arrow-clockwise.svg'
+import { ButtonWithLoader } from '../common'
+import { getGitOpsRepoConfig } from '../../services/service'
+
+export const ReloadNoGitOpsRepoConfiguredModal: FunctionComponent<ReloadNoGitOpsRepoConfiguredModalType> = ({
+    closePopup,
+    reload,
+}: ReloadNoGitOpsRepoConfiguredModalType) => {
+    return (
+        <VisibleModal2 className="confirmation-dialog">
```

Comment:
Any specific reason to choose `VisibleModal2`?

Reply:
was already there, have changed to VisibleModal